blackboxopt/optimizers/botorch_base.py

```diff
@@ -75,7 +91,7 @@ def _acquisition_function_optimizer_factory(
         "num_restarts" in kwargs
         or "raw_samples" in kwargs
         or space_has_continuous_parameters
-    ):
+    ):  # continuous AF optimization
         return functools.partial(
```

Comment:
We usually discourage inline comments, following the [PEP recommendation](https://peps.python.org/pep-0008/#inline-comments) to use inline comments sparingly, and rather comment on a new line before the affected line.
In this case that would look like
```python
        or space_has_continuous_parameters
    ):
        # continuous AF optimization
        return functools.partial(
```
or
```python
    # continuous AF optimization
    if "num_random_choices" not in kwargs and (
        "num_restarts" in kwargs
        or "raw_samples" in kwargs
        or space_has_continuous_parameters
    ):
```